qa/evil-tests/src/test/java/org/elasticsearch/threadpool/EvilThreadPoolTests.java

```diff
@@ -190,6 +191,7 @@ public void testExecutionExceptionOnScalingESThreadPoolExecutor() throws Interru
             1,
             10,
             TimeUnit.SECONDS,
+            false,
```

Comment:
I think we can check both true and false here?

```suggestion
            randomBoolean(),
```

Reply:
Sure, I changed this in ad729c90cdf439abdd2e713086bd947376988df9